documentation/development/plugin/gender.rst

```diff
@@ -0,0 +1,67 @@
+Gender plugins
+==============
+
+.. list-table::
+   :align: left
+   :stub-columns: 1
+
+   * -  Type
+     -  :py:class:`betty.ancestry.gender.Gender`
+   * -  Repository
+     -  :py:class:`betty.ancestry.gender.GENDER_REPOSITORY`
```

Comment:
Not true! This must be Project.genders. Correct for other plugin types too, in separate PRs!